### ISSUE TITLE
System.Runtime.InteropServices.COMException at EnvDTE._DTE.get_Solution()

### ISSUE THREAD

**Issue by alanhamlett** [wakatime/visualstudio-wakatime#25]
arfmach wrote at https://visualstudiogallery.msdn.microsoft.com/ca0ea1f3-e824-4586-a73e-c8e4a65323d8/view/Discussions:

This is happening a lot and making VS2013 Community crash and be forced to close.

```
Exception Type       : System.Runtime.InteropServices.COMException
Exception Message    : O filtro de mensagens indicou que o aplicativo está ocupado. (Exceção de HRESULT: 0x8001010A (RPC_E_SERVERCALL_RETRYLATER))
Exception Stack Trace:    at EnvDTE._DTE.get_Solution()
   at WakaTime.WakaTimePackage.GetProjectName()
   at WakaTime.WakaTimePackage.SendHeartbeat(String fileName, Boolean isWrite)
   at WakaTime.WakaTimePackage.<>c__DisplayClass3.<HandleActivity>b__2()
   at System.Threading.ThreadHelper.ThreadStart_Context(Object state)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.ThreadHelper.ThreadStart()
Exception Error Code : -2147417846
Exception Data   : [EmptyOrNull]
```

**Comment by gandarez** [comment]
Made pull request #26

**Comment by alanhamlett** [comment]
Just released v5.0.5 of the extension. Can someone confirm it's fixed after updating?

**Comment by gandarez** [comment]
@alanhamlett I already did some tests and everything is working as expected. Let's wait waht community says.
